torpor

```diff
@@ -7,16 +7,42 @@ type docker >/dev/null 2>&1 || { echo >&2 "Can't find docker command."; exit 1;
 
 echo "" > parameters.json
 echo "" > torpor.log
-if [ ! -f base.json ] ; then
-  echo "Cannot find base.json file"
+
+ARGS=()
+base_filename="base.json"
+output_filename="parameters.json"
+while (($#)); do
+    if [ "$1" == "--base-filename" ]; then
+        shift
+        base_filename="$1"
+    elif [ "$1" == "--output-filename" ]; then
+        shift
+        output_filename="$1"
+    else
+        ARGS+=("$1")
+    fi
+shift
+done
```

Comment:
this would be better handled by [`getopts`](http://wiki.bash-hackers.org/howto/getopts_tutorial)